recipes/qt-advanced-docking-system/build.sh

```diff
@@ -14,3 +14,4 @@ cmake \
   -DADS_VERSION=${PKG_VERSION} \
   ..
 make install -j${CPU_COUNT} 
+rm -r ${PREFIX}/lib/license
```

Comment:
```suggestion

```

Reply:
I found that a folder with license files is produced by cmake install. This folder contains the same files for `qt6-advanced-docking` and `qt5-advanced-docking`. I will soon fix the unix build script.

Reply:
nice fix!